src/pencilkit.cs

```diff
@@ -97,12 +115,20 @@ interface PKCanvasView : PKToolPickerObserver {
 		[Export ("drawingGestureRecognizer")]
 		UIGestureRecognizer DrawingGestureRecognizer { get; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Use 'drawingPolicy' property instead.")]
```

Comment:
no need for the `[Introduced]` it's already on the type as `[iOS (13,0)]` and the message does not make sense for the introduction